content/en/guides/os-commercial-ecosystem.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Creating an Open Source Commercial Ecosystem
+---
+
+An open source project is successful when it has a thriving ecosystem of companies and products built around it.  So what exactly is an open source commercial ecosystem and how can organizations help create and support sustainable open source projects? There are concrete ways to create confidence in a project’s long-term viability, encourage companies to create commercial products and services on top of open source projects, and re-invest in them with contributions, content, and other resources. These practices for enabling ecosystem development help set your projects up for success.
+
+&nbsp;
+
+**Table of Contents**
+
+- [Defining and creating a  sustainable open source commercial ecosystem](#defining-and-creating-a-sustainable-open-source-commercial-ecosystem)
+- [Ingredients of open source ecosystem success](#ingredients-of-open-source-ecosystem-success)
+- [How to build an open source ecosystem](#how-to-build-an-open-source-ecosystem)
+- [Example: Google Open Source Office](#example-google-open-source-office)
+
+&nbsp;
+
+# Defining and creating a sustainable open source commercial ecosystem
+
+Open Source Program Offices can interact with open source in two different ways: 
+
+* Outbound processes: contributing to existing or releasing new open source projects
+* Inbound processes: consuming open source projects
+
+
+When companies (and their OSPOs) work on a strategy to build their products and services on open source, they have an incentive to re-invest resources into those projects, which leads to the creation of open source commercial ecosystems. For projects to achieve this level of sustainability, companies must first have enough confidence in a project’s future that they are willing to build commercial dependencies on it.
```

Comment:
The incentive is not inherent in building on open source. Decades of "the free-rider problem" have shown us that.

Suggestion: A critical element of any business or product strategy that includes the use of open source software is the reinvestment of resources into the projects on which that strategy relies. This can lead to the creation of open source commercial ecosystems, which contribute to the viability and long-term sustainability for those projects. However, before a company will invest resources it must first have confidence in a project's future prospects such that they're willing to build commercial dependencies upon it.